middlewares/validators/flag.js

```diff
@@ -0,0 +1,25 @@
+const joi = require("joi");
+
+const addFeatureFlag = async (req, res, next) => {
+  const schema = joi
+    .object()
+    .strict()
+    .keys({
+      title: joi.string().required(),
+      enabled: joi.boolean().required(),
+      roles: joi.array().items(joi.string()).required(),
```

Comment:
Why is this required? I can see a case for us adding feature flag to individual usernames/ids, instead of using roles